src/it/scala/uk/gov/hmrc/support/ScalaCheckUtils.scala

```diff
@@ -3,25 +3,29 @@ package uk.gov.hmrc.support
 import org.scalacheck.Prop
 import org.scalacheck.Prop.collect
 
+import scala.collection.compat.immutable.LazyList
+import scala.collection.compat.immutable.LazyList.#::
```

Comment:
Need this explicitly or else the compiler tries to use `Stream.#::` from `scala.Predef`